src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml

```diff
@@ -5,7 +5,7 @@ services:
   prestashop.core.util.helper_card.documentation_link_provider:
     class: 'PrestaShop\PrestaShop\Core\Util\HelperCard\DocumentationLinkProvider'
     arguments:
-      - '@=service("prestashop.adapter.legacy.context").getContext().language.iso_code'
+      - '@=service("prestashop.adapter.legacy.context").getLanguage().iso_code'
```

Comment:
how about something like:

```
@=service("prestashop.adapter.legacy.context").getLanguage() ? @=service("prestashop.adapter.legacy.context").getLanguage().iso_code : 'en-US'
```

In the console context, the Shop is not supposed to be loaded. Instead of loading the Shop step by step in the console, I'd say we make the Console more independant from the Web state.

Reply:
interesting, except `getLanguage` always return an object
I'm gonna try something similar

Reply:
It works! thanks
This way the PR modifies much less things (I left a few bug fixes nevertheless)